src/components/display/fewlines/Input.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+export const Input = styled.input`
+  background: ${({ theme }) => theme.colors.backgroundContrast};
+  border: 0.1rem solid #a1a4b1;
+  border-radius: ${({ theme }) => theme.radii[0]};
+  height: 4rem;
+
+  ::placeholder {
+    color: #8b90a0;
```

Comment:
You should use a color from the theme here. You can even create a new theme color, called `placeholder` 😁.

Reply:
OK fixed d40bcb1